app/src/main/java/org/pesmypetcare/mypetcare/services/user/UserManagerAdapter.java

```diff
@@ -89,69 +101,95 @@ private void assignImageFromServer(User user) throws MyPetCareException {
 
     @Override
     public boolean userExists(User user) {
-        UserData userData = null;
+        AtomicReference<UserData> userData = new AtomicReference<>();;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5635133)